svn/common.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, url_or_path, type_, *args, **kwargs):
         self.__trust_cert = kwargs.pop('trust_cert', None)
 
         if type_ not in (svn.constants.LT_URL, svn.constants.LT_PATH):
-            raise ValueError("Type is invalid: %s" % (type_))
+            raise SvnException("Type is invalid: %s" % type_)
```

Comment:
Here I think a `ValueError` is appropriate, since it is actually a wrong value passed by the user of the library.